ci/docker/builder/cpu/ubuntu22.04/arm64/Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN apt update \
     && cd /usr/bin \
     && unlink gcc && ln -s gcc-12 gcc \
     && unlink g++ && ln -s g++-12 g++ \
+    && rm -rf /usr/lib/gcc/aarch64-linux-gnu/11 \ #Installed by default on ubuntu22.04, need to remove to fetch latest SVE header.
```

Comment:
should we do this change for both arm and x86 ?

Reply:
In x86 manual inclusion of header file is not done hence it won't make a difference.

Reply:
@cydrain Still CI issue is there, I have logged the path from where it tries to fetch sve header in the build part,

-- ARM SVE headers found at: /usr/lib/gcc/aarch64-linux-gnu/11/include

It is still gcc-11

Reply:
@adarshs1310 I have updated the docker image (#983), you can rebase and try e2e-arm again
if any problem, please let me know

Reply:
@cydrain arm is still failing with the same reason

-- ARM SVE headers found at: /usr/lib/gcc/aarch64-linux-gnu/11/include

This is happening because you haven't removed gcc-11 folder in your new commit. It is updating the alternative correctly but there will be presence of two gcc folders on the machine 11 and 12.

Reply:
Let me check

Reply:
For SVE we need to include the header file manually in cmake , if two headers are present it will pick the first one only no matter the update alternative settings.